public/script.js

```diff
@@ -1524,6 +1524,18 @@ function getCharacterSource(chId = this_chid) {
         return `https://pygmalion.chat/${pygmalionId}`;
     }
 
+    const githubRepo = characters[chId]?.data?.extensions?.github_repo;
+
+    if (githubRepo) {
+        return `https://github.com/${githubRepo}`;
+    }
+
+    const SourceUrl = characters[chId]?.data?.extensions?.source_url;
```

Comment:
consider proper `camelCasing` for local variables.

Reply:
fixed in [`cd95028` (#2255)](https://github.com/SillyTavern/SillyTavern/pull/2255/commits/cd950287a3617363985d08152a0a8d668f0f1db5)